build.gradle

```diff
@@ -115,6 +143,26 @@ uploadArchives {
             }
         }
     }
+}*/
+
+publishing {
+    publications {
+        mavenPublication(MavenPublication) {
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
+            groupId "$groupId"
+            artifactId 'gradle-scala-multiversion-plugin'
+            version "$version"
+            pom.withXml {
+                def root = asNode()
+                root.appendNode('description', 'Gradle plugin to build a project against multiple versions of scala')
+                root.appendNode('name', "$artifact_group:$artifact_name")
+                root.appendNode('url', 'https://github.com/ADTRAN/gradle-scala-multiversion-plugin')
```

Comment:
Do we have to do XML manipulation here like this? Can these three fields go in `pomConfig` and be set using the DSL?

Reply:
Looking at [MavenPublication](https://docs.gradle.org/current/dsl/org.gradle.api.publish.maven.MavenPublication.html), using `description`, `name`, or `url` directly isn't documented, so I'm inclined to leave it as is.